packages/components/src/components/link/index.ts

```diff
@@ -0,0 +1,12 @@
+import Link from './link.component';
+import { TAG_NAME } from './link.constants';
+
```

Comment:
This component has a dependency of `mdc-icon` and that component has to be imported over here.
```suggestion
import '../icon';

```

Reply:
Updated